### DISCUSSION TITLE
Combined microstructure

### DISCUSSION THREAD

**Discussion by LiZa111111111** [neperfepx/neper#599] [Q&A] [answered]
Hello,
Question 1 
I would like to know if there is a possibility to apply the combined orientations for the combined microstructures, see the picture below. In my case, I applied the random orientations to the equiaxed grain structure part and the textured orientation (001) to the columnar grain structure part. It was done manually, though. I would prefer if NEPER could automatically set the textured orientation for a specific part of the RVE. Is it possible?
Question 2
I have difficulties to direct the number of columnar grains. Even if I use 6000 grains, it generates only approx. 100 grains for the columnar part, see the picture below. If I would like to make a statistically representative structure, the columnar part should include more grains than that. How can I manage it?
Question 3
The sharp shaped columnar grains might be difficult to analyze, see the picture below. I have used -reg 1 option but it does not help in this case. Is there some way of reducing or controlling the number of sharp thin columnar grain generation?
 
![comb_ipf](https://user-images.githubusercontent.com/113169470/199692107-bffdaa6c-e77e-4554-9345-38fe53542606.png)

**Comment by rquey** [comment, marked as answer]
I won't be of much help here. I assume you are using a custom series of Neper commands, and it is only by modifying these commands that you can "improve" your microstructure.

**Comment by MROHOMAN** [comment]
@LiZa111111111  how did you generate the RVE? Could you please share the commands?
